tests/debugger/oegdb/commands.gdb

```diff
@@ -119,19 +119,53 @@ commands 6
     python import gdb_sgx_plugin
 
     python gdb.parse_and_eval("TCS_GSBASE_OFFSET = " + str(gdb_sgx_plugin.TCS_GSBASE_OFFSET))
-    python gdb.parse_and_eval("TD_CALLSITE_OFFSET = " + str(gdb_sgx_plugin.TD_CALLSITE_OFFSET))
-    python gdb.parse_and_eval("CALLSITE_OCALLCONTEXT_OFFSET = " + str(gdb_sgx_plugin.CALLSITE_OCALLCONTEXT_OFFSET))
-
-    python gdb.parse_and_eval("OCALLCONTEXT_LENGTH  = " + str(gdb_sgx_plugin.OCALLCONTEXT_LENGTH))
-    python gdb.parse_and_eval("OCALLCONTEXT_FORMAT[0] = '" + gdb_sgx_plugin.OCALLCONTEXT_FORMAT[0] + "'")
-    python gdb.parse_and_eval("OCALLCONTEXT_FORMAT[1] = '" + gdb_sgx_plugin.OCALLCONTEXT_FORMAT[1] + "'")
-    python gdb.parse_and_eval("OCALLCONTEXT_RBP = " + str(gdb_sgx_plugin.OCALLCONTEXT_RBP))
-    python gdb.parse_and_eval("OCALLCONTEXT_RET = " + str(gdb_sgx_plugin.OCALLCONTEXT_RET))
 
     python print("Debugger contract serialized on enclave side.")
     continue
 end
 
+# Assert that stack has been stitched correctly
+b host.c:84
+commands 7
+    python print("\n\n\nWalking ocall stack....\n\n")
+    # Read magic variable
+    set $magic_value=magic_value
+
+    # Set the magic variable in host_function.
+    set host_function_magic=$magic_value
+
+    # Walk the stack until the enclave function.
+    # This asserts ocall stack stitching.
+    set $i = 50
```

Comment:
Consider adding a comment for why 50 is your upper bound here and for the ecall stack walk.

Reply:
Fixed. Used a MAX_FRAMES variable.